### ISSUE TITLE
Ubuntu Studio 20.10 Changes (Heads-Up!)

### ISSUE THREAD

**Issue by eeickmeyer** [hakuna-m/wubiuefi#234]
Hi there, Ubuntu Studio lead here.

Great job keeping this project maintained! I just wanted to let you know that Ubuntu Studio, for 20.10 due out in October, has changed to KDE Plasma as the desktop environment and Calamares as the installer. I'm not 100% sure if this will cause an issue, but I thought I'd let you know anyhow in case it affects Wubiuefi.

Cheers!

**Comment by hakuna-m** [comment]
Thanks for your information. Currently, Calamares does not work with Wubiuefi. Currently, Wubiuefi supports only Ubiquity.

**Comment by eeickmeyer** [comment]
If that's the case, how do you support Lubuntu? Lubuntu has been on Calamares since 18.10 and seems to work, and you also mention it's supported.

**Comment by hakuna-m** [comment]
> Lubuntu has been on Calamares since 18.10 and seems to work, and you also mention it's supported.

Lubuntu 18.10: #121 I disabled Lubuntu configuration.
Lubuntu 19.04: #149 I added Lubuntu configuration again because somebody asked me for that configuration to run live environment and further tests. I added it as known issue to the release notes.
Lubuntu 19.10, 20.04: Probably, because of an copy/paste mistake I forgot the known issue in the release notes.
Lubuntu 20.04.1: #232 I added it to the release notes again. I re-tested it yesterday. So I can confirm the issue is still there.

**Comment by eeickmeyer** [comment]
Ok, that's fair. I highly recommend researching a way to implement Calamares support, if that's the case. Lubuntu has been on it for nearly 2 years, we (Ubuntu Studio) are changing starting with 20.10, and Kubuntu will likely change as soon as Calamares has a minimal install option.

Our rationale is this: the Qt form of the Ubiquity installer is hard-coded to be Kubuntu-only. Unfortunately, getting any merge requests into Ubiquity is nearly impossible within a certain testing timeframe as everything has to go through the Ubuntu Foundations team, and they have "better things to do" than to re code Ubiquity for the flavors when their focus is on Ubuntu (proper) Desktop. So, rather than deal with bureaucracy, the leader of Lubuntu decided to switch to Calamares for 18.10 since they were switching to a Qt-based desktop environment at the time (LXQt). With Ubuntu Studio switching to KDE Plasma, we decided it would be best if we didn't have the Kubuntu branding in our installer (how confusing would that be), so we found ourselves in the same situation as Lubuntu from 2 years ago. Calamares seemed like not only the logical choice, but the best-tested one.

So, basically this issue has been around for the past two years, and with more official flavors of Ubuntu adopting Calamares, I think it's probably time to consider working on this, otherwise wubiefi will end up just like wubi of years ago. Lubuntu, Ubuntu Studio, and likely Kubuntu will have this issue, along with unofficial derivatives working on their official status, such as Ubuntu Cinnamon and Ubuntu Unity, both of which are using Calamares. I understand this has been a low priority since only one flavor was affected.

I would help with this, but I'm not a coder by any stretch of the imagination. I'm a packager and leader, but I have no coding chops to really speak of.

**Comment by hakuna-m** [comment]
Some weeks ago, we tested an installer independend script, which should do the job of the small part of Ubiquity which Wubiuefi really needs. The results of the tests were good. So I hope we can release it with 20.10 as alternate for installer issues.

Calamares support would be also an option, but as Wubiuefi still needs Ubiquity for Ubuntu, it does not solve issues occuring with Ubuntu.
